app/src/components/pages/searchPage/searchPage.tsx

```diff
@@ -0,0 +1,113 @@
+"use client";
+import {
+  Box,
+  Text,
+  Pagination,
+  Heading,
+  Flex,
+  HorizontalRule,
+  TagSet,
+  Link,
+  Icon,
+} from "@nypl/design-system-react-components";
+import React from "react";
+import { CARDS_PER_PAGE } from "@/src/config/constants";
+import { displayResults } from "@/src/utils/utils";
+import Filters from "../../search/filters";
+import { CardsGrid } from "../../grids/cardsGrid";
+
+const SearchPage = ({ data }) => {
+  const totalPages = 10;
+  return (
+    <Box id="mainContent">
+      <Box
+        sx={{
+          background: "ui.bg.default",
+          padding: "l",
+          marginBottom: "m",
+        }}
+      >
+        <Box
+          maxWidth="1280px"
+          mx="auto"
+          sx={{ paddingLeft: { base: 0, xl: "s" } }}
+        >
+          <Heading
+            size="heading2"
+            sx={{
+              maxWidth: "1250px",
+              marginBottom: "m",
+            }}
+          >
+            {`Displaying ${displayResults(data.numResults, CARDS_PER_PAGE, 1)}
+                    results for "keyword"`}
+          </Heading>
+          <Filters />
+        </Box>
+      </Box>
+      <Box
+        maxWidth="1280px"
+        mx="auto"
+        sx={{ paddingLeft: { base: 0, xl: "s" } }}
+      >
+        <HorizontalRule />
+        <Flex alignContent="center" gap="xs">
+          <Text size="subtitle2" sx={{ margin: 0, fontWeight: 400 }}>
+            Filters applied:
+          </Text>
+          <TagSet
+            isDismissible
+            id="search-filter-tags"
+            onClick={() => {}}
+            tagSetData={[
+              { id: "audio", label: "Audio" },
+              { id: "video", label: "Video" },
+            ]}
+            type="filter"
+          />
+        </Flex>
+        <HorizontalRule />
+        <Heading marginTop="xl" size="heading5">{`Displaying ${displayResults(
```

Comment:
Wrap this in a div with `aria-live` or add the attr to Heading. Or add `tabIndex={-1}` to programmatically focus on the heading after it gets updated.